src/components/Provider.js

```diff
@@ -1,8 +1,10 @@
-import { Component, Children } from 'react'
+import React, { Component, Children } from 'react'
 import PropTypes from 'prop-types'
 import { storeShape, subscriptionShape } from '../utils/PropTypes'
 import warning from '../utils/warning'
 
+const prefixUnsafeLifecycleMethods = parseFloat(React.version) >= 16.3
```

Comment:
@elyobo @timdorr Thanks for fixing this, but there's still one more issue here. `parseFloat("16.10.1")` parses as 16.1 which is of course less than 16.3, so this fails on the latest release of React.

Reply:
Ah, that's not good... Can you work up a fix PR?

Reply:
Actually, I got it: #1407

Reply:
Ha, we've gone back to a feature check like this PR originally shipped with before https://github.com/reduxjs/react-redux/pull/1383#issuecomment-524483129 :D  Thanks for the pick up Sam.